docker/development.yml

```diff
@@ -29,6 +29,7 @@ services:
         - SEARCH_URL=${SEARCH_URL}
         - FEED_DISCOVERY_URL=${FEED_DISCOVERY_URL}
         - STATUS_URL=${STATUS_URL}
+        - DEPENDENCY_DISCOVERY_URL=${DEPENDENCY_DISCOVERY_URL}
```

Comment:
You don't need the `=...`, just forward the value: 

```
- DEPENDENCY_DISCOVERY_URL
```

If you want to fix the other ones above that do this, feel free.